tableauserverclient/models/connection_credentials.py

```diff
@@ -0,0 +1,5 @@
+class ConnectionCredentials(object):
+    def __init__(self, name, password, embed=True):
+        self.password = password
```

Comment:
if I fix order of property set in credentials do you want me to fix auth while I am doing it as I copied the code for this class from auth. New code would be

`class TableauAuth(object):
    def __init__(self, username, password, site='', user_id_to_impersonate=None):
        self.username = username
        self.password = password
        self.site = site
        self.user_id_to_impersonate = user_id_to_impersonate
`